src/main/java/com/denizenscript/denizencore/scripts/commands/core/MongoCommand.java

```diff
@@ -0,0 +1,765 @@
+                scriptEntry.addObject("parameter", arg.asElement());
+            }
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesPrefix("command_map", "cmd_map")) {
+                scriptEntry.addObject("action", new ElementTag("command_map"));
+                scriptEntry.addObject("command_map", arg.object);
+            }
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesPrefix("find")) {
+                scriptEntry.addObject("action", new ElementTag("find"));
+                scriptEntry.addObject("filter", arg.object);
+            }
+            else if (!scriptEntry.hasObject("find_by_id")
+                    && arg.matchesPrefix("by_id", "find_by_id")) {
+                scriptEntry.addObject("find_by_id", arg.asElement());
+            }
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesPrefix("insert", "i")) {
+                scriptEntry.addObject("action", new ElementTag("insert"));
+                scriptEntry.addObject("insert_data", arg.object);
+            }
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesPrefix("update", "u")) {
+                scriptEntry.addObject("action", new ElementTag("update"));
+                scriptEntry.addObject("update", arg.object);
+            }
+            else if (!scriptEntry.hasObject("new")
+                    && arg.matchesPrefix("new")) {
+                scriptEntry.addObject("new", arg.object);
+            }
+            else if (!scriptEntry.hasObject("upsert")
+                    && arg.matchesPrefix("upsert")
+                    && arg.asElement().isBoolean()) {
+                scriptEntry.addObject("upsert", arg.asElement());
+            }
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesPrefix("use_database", "use_db")) {
+                scriptEntry.addObject("action", new ElementTag("use_database"));
+                scriptEntry.addObject("new_database", arg.asElement());
+            }
+            else if (!scriptEntry.hasObject("action")
+                    && arg.matchesPrefix("use_collection", "use_col")) {
+                scriptEntry.addObject("action", new ElementTag("use_collection"));
+                scriptEntry.addObject("new_collection", arg.asElement());
+            }
+            else {
+                arg.reportUnhandled();
+            }
+        }
+        if (!scriptEntry.hasObject("id")) {
+            throw new InvalidArgumentsException("Must specify an ID!");
+        }
+        if (!scriptEntry.hasObject("action")) {
+            throw new InvalidArgumentsException("Must specify an action!");
+        }
+        if (!scriptEntry.hasObject("upsert")) {
+            scriptEntry.defaultObject("upsert", new ElementTag("false"));
+        }
+    }
+
+    @Override
+    public void execute(final ScriptEntry scriptEntry) {

```

Comment:
Use the new autoExecute, don't PR new commands using old syntaxes (this is actually two entire command registration formats behind here - the newer arg handler was the `argAsX` system, and autoExecute is newest)

Reply:
Also, need a CoreConfiguration option to disable mongo - should be disabled by default even
Same as redis/sql have